knowledge_base/python_decorator/README.md

```diff
@@ -0,0 +1,78 @@
+# Python decorator
+
+This example shows how to package a **Python decorator–based job** so you can
+call regular Python functions from a Databricks job with minimal boilerplate.
```

Comment:
Can you make it more clear this is about Python support for DABs?

E.g. include a doc link in this paragraph.